vuln/core/14.json

```diff
@@ -10,5 +10,5 @@
   "affectedEnvironments": [
     "all"
   ],
-  "severity": "unknown"
+  "severity": "critical"
```

Comment:
Although the Github Advisory points it as CRITICAL, the Node.js team was dealing with it as HIGH 

Private ref: https://github.com/nodejs-private/node-private/commit/0b76d02974eb46ef2eea6639657c9d0bf72095cb

```
Semver Minor:

* openssl:
  - Upgrade to 1.0.2i, fixes a number of defects impacting Node.js:
    https://github.com/advisories/GHSA-7c6f-r892-6p89 ("OCSP Status Request extension unbounded memory
    growth", high severity), https://github.com/advisories/GHSA-w2rw-pv8p-h9c8, https://github.com/advisories/GHSA-w6xc-jcff-g3vg,
```